src/quantuminspire/qiskit/backend_qx.py

```diff
@@ -188,7 +184,13 @@ def get_experiment_results(self, qi_job):
             histogram = QuantumInspireBackend.__convert_histogram(result, measurements, result['number_of_qubits'],
                                                                   job['number_of_shots'])
             histogram_obj = Obj.from_dict(histogram)
-            experiment_result_data = ExperimentResultData(counts=histogram_obj)
+            raw_data_url = result.get('raw_data_url')
```

Comment:
Would be nice to hide this in the api, like `api.get_raw_data(result_id)` or `api.get_raw_data_from_result(result_id)`